pkg/azure/gw-machineset.go

```diff
@@ -0,0 +1,83 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package azure
+
+var machineSetYAML = `apiVersion: machine.openshift.io/v1beta1
```

Comment:
you can use `go:embed` directive and then read it as string/file. See for eg  https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/gather/summary.go#L40